Power Up Proto/src/org/usfirst/frc/team5493/robot/commands/TransmissionReverse.java

```diff
@@ -7,12 +7,12 @@
 /**
  *
  */
-public class SolenoidReverse extends Command {
+public class TransmissionReverse extends Command {
 
 	boolean isFinished;
 	
-    public SolenoidReverse() {
-    	requires(Robot.solenoid);
+    public TransmissionReverse() {
+    	requires(Robot.throwDaggersInBensEyes);
     	isFinished = true;
         // Use requires() here to declare subsystem dependencies
         // eg. requires(chassis);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)